<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="access_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>350</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>클라 접속 여부</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>560</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>클라 요청</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="access_store">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>350</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>매장 클라 접속 여부</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>클라 종료 여부</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="disconnect_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>350</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
